arborist/server.go

```diff
@@ -226,6 +226,19 @@ func (server *Server) handleAuthMappingGET(w http.ResponseWriter, r *http.Reques
 	usernameQS, ok := r.URL.Query()["username"]
 	if ok {
 		username = usernameQS[0]
+	} else if authHeader := r.Header.Get("Authorization"); authHeader != "" {
+		// Fall back to JWT for username. Added for Arborist UI integration...
+		server.logger.Info("No username in query args; falling back to jwt...")
+		userJWT := strings.TrimPrefix(authHeader, "Bearer ")
```

Comment:
is there not an existing utility function that does this?

Reply:
That decodes the jwt and extracts the username? I don't think so? Maybe you are thinking of just `server.decodeToken`, called below this?

Reply:
Or if you mean trimming the header, no. Before this, trimming the "bearer" out of the token happened in one place. Now it happens in two places, and the second place is this derpy business

Reply:
ah, okay, no worries.